packages/vitest/src/node/cli-api.ts

```diff
@@ -17,7 +17,7 @@ export interface CliOptions extends UserConfig {
 export async function startVitest(cliFilters: string[], options: CliOptions, viteOverrides?: ViteUserConfig) {
   process.env.TEST = 'true'
   process.env.VITEST = 'true'
-  process.env.NODE_ENV ??= options.mode || 'test'
+  process.env.NODE_ENV = options.mode || 'test'
```

Comment:
Why this? `process.env.NODE_ENV` should not be overwritten, if it was defined by user

Reply:
Upps! I misunderstood the parts to be corrected.
Sorry😰 Close this PR.

Reply:
Thanks @sheremet-va 
I'll reopen 😃